terraform/apigateway.tf

```diff
@@ -0,0 +1,21 @@
+data "template_file" "coepi_api_swagger" {
```

Comment:
Where do we set it to be edge-optimized ?

Reply:
Not sure but if it's something that the AWS API exposes we can do it.

Reply:
I didn't see this in the original manual steps but I can create a followup issue.

Reply:
#5 done